roles/openshift_facts/defaults/main.yml

```diff
@@ -130,4 +130,12 @@ openshift_node_groups:
     labels:
       - 'node-role.kubernetes.io/compute=true'
     edits: []
+  - name: node-config-master-infra
+    labels:
+      - 'node-role.kubernetes.io/infra=true,node-role.kubernetes.io/master=true'
+    edits: []
+  - name: node-config-all-in-one
```

Comment:
I agree with @vrutkovs let's update the papr inventories to utilize this for CI.